### DISCUSSION TITLE
Why when I activate vsyn the fps is limited to 40 fps?

### DISCUSSION THREAD

**Discussion by sawa-ko** [ppy/osu#19579] [Q&A] [answered]
I have a monitor that supports 60hz but when I activate the vsync option in osu the fps are limited from 35 to 40 fps, I don't understand why.

https://user-images.githubusercontent.com/56084970/182918986-e8de2518-ce32-4c1d-98b5-8db1122b1252.mp4

**Comment by peppy** [comment, marked as answer]
This means the game can't hold a stable 60fps and is switching between 30 and 60, averaging to somewhere in between.

**Reply by sawa-ko** [reply]
but why can not reach 60 fps? because if I activate the x2 refresh rate option it reaches 120fps without problems.

**Reply by peppy** [reply]
It's a matter of individual frame timings. A single slow frame can cause vsync to switch back to 30fps. It's up to the drive to decide.

If you want a better vsync experience, please make sure you use gsync or freesync at a driver/display level.
